file-generating/ronn/README.md

```diff
@@ -0,0 +1,162 @@
+
+<br>
+
+Jump to:
+ - [Downloading the container](#downloading-the-container).
+ - [Adding the container to your PATH](#adding-the-container-to-your-path).
+ - [Usage](#usage).
+ - [More examples](#more-examples).
+ - [Specifying the output file](#specifying-the-output-file).
+ - [Making your own manpage](#making-your-own-manpage).
+ - [Demo](#demo).
+
+
+<br>
+
+#### What you need:
+ - Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim`, or `nano`.
+ - A markdown test file, like [this one](https://raw.githubusercontent.com/sylabs/examples/master/file-generating/ronn/manpage-test.md).
+
+
+___
+
+<br>
+
+
+### Downloading the container:
+
+For `x86_64`:
+
+```
+$ singularity pull ronn.sif library://sylabs/examples/ronn:latest
+```
+
+For `armv7l`: (`32bit`)
+
+```
+$ singularity pull ronn.sif library://sylabs/examples/ronn-arm:latest
+```
+
+<br>
+
+Verifying the container:
+
+```
+$ singularity verify ronn.sif
+Verifying image: ronn.sif
+Data integrity checked, authentic and signed by:
+        Sylabs Admin <support@sylabs.io>, KeyID EDECE4F3F38D871E
+```
+
+<br>
+
+### Adding the container to your PATH:
+
+We are going to do this without any root access, so we will install `ronn.sif` to `~/.local/bin/`:<br>
+You can skip this step if you already have `${HOME}/.local/bin/`.<br>
+Of course you can always move it to `/usr/local/bin/`, but that requires root access.
+
+First, make the `${HOME}/.local/bin/`:
+
+```
+$ mkdir -p ${HOME}/.local/bin/
+```
+
+Then, add it to your PATH
+
+```
+$ echo 'export PATH=${PATH}:${HOME}/.local/bin/' >> ~/.bashrc
+$ source ~/.bashrc
+```
+
+<br>
+
+Finaly, move `ronn.sif` to `~/.local/bin/`
+
+```
+$ mv ronn.sif ~/.local/bin/
+```
+
+<br>
+
+## Usage:
+
+For this example, let's make a manpage for this container.
+
+First, download the markdown file:
+
+```
+$ wget https://raw.githubusercontent.com/sylabs/examples/master/file-generating/ronn/ronn.sif.md
+```
+
+<br>
+
+Then, generate the manpage:
+
+```
+$ ronn.sif --man ronn.sif.md
+```
+
+We use the `--man` option to only generate the manpage, and not a `.html`, and `.gzip` file.<br>
+So if you just want a `.gzip` file, use the `--gzip` option, same for `--html`, and `--man`.
+
+Viewing the manpage:
+
+```
+$ man ./ronn.sif.1
+```
+
+<br>
+
+### More examples:
+
+Let's say you only want a `.gz`, and a `.html`, then run this:
+
+```
+$ ronn.sif --gzip --html your_markdown_file.md
+```
+
+Now you should have your `your_markdown_file.1.gzip`, and `your_markdown_file.1.html`.
+
+<br>
+
+### Specifying the output file:
+
+To specify the output file, use the `out=` function:
+
+```
+$ ronn.sif --man out=some_output_file.1 your_markdown_file.md
+```
+
+<br>
+
+### Making your own manpage:
+
+You can use the [`template-manpage.md`](https://raw.githubusercontent.com/sylabs/examples/master/file-generating/ronn/template.md) to make your own manpage, just download it with `wget`:
```

Comment:
This is a dead link

Reply:
Well, if you replace `sylabs` with `WestleyK`, and `master` to `WestleyK-ronn`, it seems to work fine for my branch, so I think it will work once it is merged... I think...